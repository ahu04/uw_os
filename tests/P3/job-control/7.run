sleep 20.0
##sleep 0.5
\x1A
##sleep 0.5
##assertStat T sleep 20.0
sleep 20.1
##sleep 0.5
\x1A
##sleep 0.5
##assertStat T sleep 20.1
sleep 20.2
##sleep 0.5
\x1A
##sleep 0.5
##assertStat T sleep 20.2
jobs
fg 3
##sleep 0.5
##assertStat S sleep 20.2
\x03
fg 2
##sleep 0.5
##assertStat S sleep 20.1
\x03
fg 1
##sleep 0.5
##assertStat S sleep 20.0
\x03
exit
