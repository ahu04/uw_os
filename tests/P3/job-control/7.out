${PROMPT}${PROMPT}${PROMPT}${PROMPT}1: sleep 20.0
2: sleep 20.1
3: sleep 20.2
${PROMPT}${PROMPT}${PROMPT}${PROMPT}